### DISCUSSION TITLE
2024-07-25 CUE Community Update

### DISCUSSION THREAD

**Discussion by dominikdm** [cue-lang/cue#3290] [Community Calls]
We are excited to announce our next CUE Community Update for Thursday, July 25, 2024 at 1500 UTC.

Please reply with any questions/topics you would like to see covered.

### Agenda
- Project updates
- Evaluator updates including performance
- CUE with other languages
- Modules and Central Registry updates
- cuelang.org and Docs and Content
- CUE LSP updates
- Encodings
- Embed Proposal
- Community updates

### Meeting details
https://meet.google.com/kre-vwuq-xpc

This session will also be recorded for those unable to join us live.

### CUE Community shared calendar
[Subscribe to the shared calendar](https://calendar.google.com/calendar/u/0?cid=Y19lNzkxMWQ5OWQ4ZGIyMmU2ZTVjMzhkMTVkNjY2ZTVlNjdiNWE5ODNkZWU4N2JmNTU2NDY3NzI1OGIxYjJhMTFhQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20).

### Recording
https://youtu.be/h-1lN_rrpyY

### Slides
https://docs.google.com/presentation/d/138sPYgiT5pxiosXD48uug-yNfrTgqcIqwMDzEZi0EUM/

### Auto-generated transcript
https://docs.google.com/document/d/1Rts6hQj0P4YLPwYA4p7H2AHSSE_vMaIml7zm1EyfVwA/

**Comment by mhemken-vts** [comment]
I'm interested in hearing about cuepls.

**Reply by dominikdm** [reply]
Great - we will talk about cuepls at the next Community Call.

**Comment by dominikdm** [comment]
Added the agenda.

**Comment by dominikdm** [comment]
Added recording, slides and transcript.
